xbmc/addons/AddonDll.h

```diff
@@ -176,6 +177,25 @@ bool CAddonDll<TheDll, TheStruct, TheProps>::LoadDll()
 #endif
   if (!XFILE::CFile::Exists(strFileName))
   {
+    std::string altbin = CSpecialProtocol::TranslatePath("special://xbmcaltbinaddons/");
+    if (!altbin.empty())
+    {
+      strAltFileName = altbin + m_props.libname;
```

Comment:
yeah, that was the intention. But the case /some/path/<addon_dir>/<addon_lib> is also checked afterward.